### ISSUE TITLE
9/16 EBPsy and EBPharm course data

### ISSUE THREAD

**Issue by holbrooa** [lzim/teampsd#462]
**ORIGINATOR**
*Please follow the steps below to open an issue for the MTL Workgroups to address. Be sure to provide a descriptive title for the issue when saving.*

**Step 1** - Please assign this issue to the issue_tracker, under the “Projects” menu on the right side of your GitHub page.

**Step 2** - Please answer the questions below by placing a lowercase "x" between the applicable brackets below and providing brief narratives and screenshots where appropriate:

1. Is this issue a problem, such as a malfunction in the simulation, an error in documentation or similar concern, or is this a new idea
   we should consider?
   
   - [ ] Problem 
   - [x ] New idea
   
2. What team are you on (exg., Menlo Park, Team Blue)?

3. How urgent is this issue?

   - [x] Urgent - *should be addressed right away!*
   - [ ] Non-urgent - *can be prioritized within normal workstreams.*

4. In the space provided below, please describe the information or behavior you expected from the product or otherwise describe what you expected to see or hear. 

Need to add course/dose data to comparator for 6 HF templates. See table of EBP completer defs for ACT, PE, CPT, IBCT, IPT, CBT-D (it's different for each).

And also use scripts and tables from Jodie's team for EBPharm dose.

5. Please provide a screenshot of the bug, if possible.  Make as many screenshots as you like that would be helpful to illustrate the issue.

   *<< Paste screenshots here>>*

6.  Within which product did you notice the problem? *Check all that apply*

**Data Products**

   - [ ] SharePoint
   - [x ] Data/Queries
   - [ ] Team Data
   - [ ] R-reports
   
**Instructional Products**
  
   - [ ] See Guide
   - [ ] Say Guide
   - [ ] Videos
   - [ ] Post tests
   - [ ] Fidelity Checklist
   - [ ] Cheatsheets
   - [ ] Other?  Please describe here >>
   
**Simulations**
   
   Home
   - [ ] Login 
   - [ ] Session
   - [ ] Experiment Maintenance Tile
   - [ ] Team Data Menu Maintenance
   - [ ] Chat
   - [ ] Help
   - [ ] Facilitator Dashboard
   
   Care Coordination
   - [ ] Model Diagram
   - [ ] Outputs and Text
   - [ ] Experiments
   - [ ] Export function
   - [ ] Save function
   - [ ] Run/reset function
   - [ ] "i" Information
   - [ ] Team Data Table
   
   Medication Management
   - [ ] Model Diagram
   - [ ] Outputs and Text
   - [ ] Experiments
   - [ ] Export function
   - [ ] Save function
   - [ ] Run/reset function
   - [ ] "i" Information
   - [ ] Team Data Table
   
   Psychotherapy
   - [ ] Model Diagram
   - [ ] Outputs and Text
   - [ ] Experiments
   - [ ] Export function
   - [ ] Save function
   - [ ] Run/reset function
   - [ ] "i" Information
   - [ ] Team Data Table
   
   Aggregate 
   - [ ] Model Diagram
   - [ ] Outputs and Text
   - [ ] Experiments
   - [ ] Export function
   - [ ] Save function
   - [ ] Run/reset function
   - [ ] "i" Information
   - [ ] Team Data Table
   
   Suicide Prevention
   - [ ] Model Diagram
   - [ ] Outputs and Text
   - [ ] Experiments
   - [ ] Export function
   - [ ] Save function
   - [ ] Run/reset function
   - [ ] "i" Information
   - [ ] Team Data Table
   
7. What kind of device are you using?
   - [ ] Tablet
   - [ ] Smartphone
   - [ ] PC

8. What browser were you using?
   - [ ] Chrome  *What version?*
   - [ ] Edge    *What version?*
   - [ ] Safari  *What version?*

**<< If you are highlighting a new idea, please continue >>**

9. Please describe your idea below.  Feel free to provide a scan or screen capture of a sketch of your idea.

10. Why is this idea important, in terms of your work?

11. How is this idea related to a problem you face in your daily practice?

12. Is there a deadline or other scheduled constraint driving your request that we should be aware of? *This will help us to prioritize
    this request.*

**THANK YOU!!**

---------------------------------------------------------------------------------------------------------------------------------------
**STOP HERE. INFORMATION BELOW IS TO BE FILLED OUT BY WORKGROUPS AS PART OF TRIAGE AND WORK BREAKDOWN PROCESS**
---------------------------------------------------------------------------------------------------------------------------------------

**Executive Summary**
Need to add course/dose data to comparator for 6 HF templates. See table of EBP completer defs for ACT, PE, CPT, IBCT, IPT, CBT-D (it's different for each).
*Estimated person-hours to complete:*
40
*Estimated date for completion:*
8/2/19
*Lead team for effort:*
quant
*Key people (use @ assignment for people whose input will be necessary):*
@holbrooa @saveth @lzim 

**Step 1 - Complete the Interdependencies Table**

Workgroup Leads review information provided by originator and collaborate to identify interdependencies. Then they describe modifications that are needed in the appropriate column and estimate hours needed to make changes:
 
● *Workgroup Products* - This column identifies the workgroups and the products for which they are responsible. This column can be
  modified by a workgroup lead on an ad-hoc basis to add one-time products or can be modified to include new and enduring products.
  
● *Dependency?* - This column is used by the workgroup leads during collaboration to identify the existence of an interdependency.

● *Description of Change or Action Needed* - Workgroup leads briefly describe the change required.

● *Hours* - Workgroup leads estimate the person-hours required to complete the task.

Workgroup Products        | Dependency?  | Description of Change Needed               |  Hours Estimate
------------------------- | -------------|--------------------------------------------|-----------------
**Documentation**            
Brochures                 |              |                                            |
MTL See Guide             |              |                                            |
MTL Say Guide             |              |                                            |
MTL Videos                |              |                                            |
Post Test                 |              |                                            |
**Investigators**         |              |                                            |
Presentation              |              |                                            |
Quarterly Report          |              |                                            |
Annual Report             |              |                                            |
**Qualitative**           |              |                                            |
MTL Fidelity Checklists   |              |                                            |
Systems Thinking Codebook |              |                                            |
CFIR Codebooks            |              |                                            | 
**Quantitative**          |              |                                            |
Data UI                   |              |                                            |
Data Queries              |              |                                            |
SharePoint                |              |                                            |
Excel Outputs             |              |                                            |
ModelParameters           |              |                                            |
tt Reports                |              |                                            |
**Model**
CC                        |              |                                            |
MM                        |              |                                            |
PSY                       |              |                                            |
AGG                       |              |                                            |
SP                        |              |                                            |
**Sim User Interface**    |              |                                            |
Model Diagram             |              |                                            |
"i"Information            |              |                                            |
Experiments               |              |                                            |
Outputs/Charts            |              |                                            |
Exports                   |              |                                            |
Master Crosswalk Table    |              |                                            |

**Step 2 - Identify Constraints that may affect the team’s ability to complete the necessary actions.** 

Describe the constraint in terms of capability, capacity or affect on grant research goals. If there are no constraints identified, then leave the table blank.

●  Type - The type of constraint identifies one or more resource areas that are affected.  

●  Workgroup - this is the workgroup responsible for developing a new or modifying an existing product.

●  Discussion/recommendation - A brief narrative that expands the information regardingthe constraint and ideally a recommendation for
   overcoming the constraint.

**Type Constraint (check all that apply) / Workgroup(s) Affected / Discussion/Recommendation**

- [ ] Capacity      
- [ ] Cost         
- [ ] Schedule     
- [ ] Interference 
- [ ] Capability   
- [ ] Other (describe)

**Step 3 - Identify supported milestones.** *Affix the milestone at the issue level using available GitHub milestones.*

**Step 4 - Indicate the due date for completing the actions identified above, assuming constraints are removed.**

**Step 5 - Identify the lead workgroup for the action (check one):**

- [ ] Documentation
- [ ] Investigators
- [ ] Quantitative
- [ ] Qualitative
- [ ] Model
- [ ] Sim UI

**Step 6 - Go to the top of this section and complete Executive Summary information**

Note to Workgroups:  Use the issue thread to document further discussion, alternatives development and estimates.  However, be sure to update this form if dates, team responsibility or other information changes.

**Comment by holbrooa** [comment]
I changed this issue to also apply to EBPharm dose, with the help of Jodie's resources.

**Comment by lzim** [comment]
@holbrooa and @saveth 

Do we need to discuss where this fits in our overall priority list? On Monday, I know we prioritized documentation very heavily. Let me know if we need meeting time to cover this.

Thanks!

**Comment by holbrooa** [comment]
Code is written for this, but I'm still squashing bugs.  I'm also doing my best to document more thoroughly as I go, so when the code is fully done and ready to be handed off, there will be documentation already available at the same time. That's slowing me down, obviously, so I'm not quite hitting the deadline of having this done today. I'm going to leave the due date as is, but just know that this work is "late" by virtue of the documentation-focused slowing down that we talked about, so perhaps that's not really a bad thing.

I went ahead and moved it to testing this morning. Which I guess I'm sort of using as "testing, bugfixing, and documenting" really.

**Comment by staceypark** [comment]
@holbrooa I moved the due date to the beginning following our convention of Title = due date + description.

Like we discussed at the Workgroup Leads meeting yesterday, #462, #615, and #424 are our top priority as we need them done to submit our protocols for the grants prior to launch.

You & @saveth will be at the AI conference starting 9/18 (and you have to travel out here) so we need these done by latest next Monday 9/16 as our top priority.

**Comment by saveth** [comment]
@holbrooa would you be able to provide any information on what the data structure or the column names will be like for the file you'll be providing to me? I would like to get a sense of how I should be thinking of the coding for #615. Thanks.

FYI: @lzim @staceypark

**Comment by holbrooa** [comment]
@saveth I'll end up with patient-level information tracking who hits completer status when. So what structure do we want for your analysis? Same as the initiate data, where I aggregate by sta6a and month for cy2018? That would make sense to me if it's part of the same analysis.

**Comment by saveth** [comment]
@holbrooa Since @lzim wants the EBP Dose measure to be in line with the EBP Initiate Calculation, then we do want it to be aggregated to the Sta6a level. Will this be similar to the EBPsy Initiate sheet or more like the BaseAndRx sheet? I'm assuming that the denominator will be the same, we're using the denominator from the BaseAndRx sheet that you've already provided?

**Comment by holbrooa** [comment]
I'm picturing just another column, or six columns if we want one per type of completer, on the EBPsy sheet. But will that work for what you need to do?

**Comment by saveth** [comment]
@holbrooa If you're planning on just updating the comparator data file, then a column of EBPDosePatients would be appropriate. With regards to the EBPharm Dose:

-  Depression: I'm assuming this would be the sail43h and sail 47h numerators? Is this correct?
- AUD and OUD: are these the same people as those in the Initiate or are they a separate count, therefore new variables?

**Comment by holbrooa** [comment]
@saveth I think that's correct about the depression patients. I'm not sure yet about the AUD and OUD dose (the scripts we got are formatted very differently) but I'll let you know.

**Comment by lzim** [comment]
@holbrooa and @

Very glad to see some action on this “issue” and I hope that #615 and #462 are being tracked well, despite being separated FYI: @staceypark 

> AUD and OUD: are these the same people as those in the Initiate or are they a separate count, therefore new variables?

- I’m not sure I understand the operational definition of the phrase: “...are these the same people as those in Initiate...” Seemingly what is important according to how the tests of specific aims were written is what 1) EBP Reach (Initiate) and 2) EBP Reach (Course) are for a sta6a _with monthly observations_
- So, for me the clarity that I’m looking for is how we will assess (using the updated codes you have) this with a monthly observation.
- I welcome ongoing details here, and can track and try to support your work on this while I’m traveling. 🛫

**Comment by saveth** [comment]
@lzim @holbrooa As indicated in https://osf.io/z7y8m/wiki/home/ , the definition of AUD and OUD Initiate in the grant and how we operationalize these variables are counting anyone who has a prescription for AUD or OUD. 

Given that the grant define course for EBPharm as anyone who has enough refills for a guideline-recommended trial for each medication, I do not know if this mean if they are counting the same people for Dose as Initiate or are different people? Not sure if the code @holbrooa got from Jodie help clarifies this or how this grant definition is being operationalize in the data query.

**Comment by holbrooa** [comment]
@lzim One bit of logic to double-check here, since you're monitoring:

In general, the main way EBP completers get captured is "X visits (with a template) in Y days" type of logic. For the purpose of these completers, I'm not using the concept of graduation, i.e. if you hit the threshold but continue getting more visits, that's ok, you still get credit. So then I have to "throw out" those subsequent visits, because we don't want to count a patient as having completed again in the following month. So in the CPT completer script where Jodie's team is giving credit if you get 8 visits in 180 days, I find the first time a patient hits that mark, i.e. the 8th visit in a span of 180 days or less, then throw out the visits after that so that you get counted once in the month where you "complete" but then you don't count again in the following month. E.g. a patient satisfies the above completion criterion for their visits 2 through 9, they get counted as a completer in the month that that 9th visit happened, and then I don't allow them to count again for visits 3 through 10, even though they technically satisfy the 8 visits in 180 days logic there.  I'm fairly sure this is right, but if you have thoughts, this is a good time, before I hit the button and run queries that take hours.  

tl;dr: I'm capturing completers regardless of "graduation" - so patients can continue getting templates after they've completed and I will count them once in the month where they first hit the completion criterion.

**Comment by lzim** [comment]
Right @saveth these are exactly the types of data structure and other details, that I am seeking clarity about too. 

Except that, given that the unit of observation and analysis in the grant is **sta6a** each **month...**

1.  it should be less critical whether a monthly sta6a observation for EBP Reach Initiate and EBP Reach Course are the same people for any given month.
- Albeit, at an under specified conceptual level, it would seem that the numerator for EBP Reach Initiate at a given time point, should become an EBP Reach Course denominator at a later observation. 

2.  The key information being: Given how OMHSP typically handles an EBP course numerator and denominator, how does our monthly omnibus EBP Reach Course (measurement & EBPsy Course and EBPharm Course, respectively) jive with the typical quarterly observations used in typical operations. 

Thanks @holbrooa and @saveth! 🤙🏽
FYI: @staceypark

**Comment by lzim** [comment]
@holbrooa

Our messages crossed in the ether...reading your post now on the runway...and will try to get a reply to you before I lose internet in the air ✈️

**Comment by lzim** [comment]
@saveth and @holbrooa

>  "We define reach as the proportion of patients diagnosed with OUD, AUD, PTSD, or depression (ICD-10 codes) who meet EBPsy and EBPharm 1a) initiation and 1b) course measures (numerator) divided by the total number of patients with these diagnoses (denominator) at that clinic station."

- Right. I believe your recap is accurate, and drawn from the grant. 
- But, it is just underspecified (meaning, also underspecified in the grant) for _the role of time in the EBP Course numerator/denominator_ and the issues that are discussed above by @holbrooa and I.

- Naturally, the maximum denominator is the eligible EBP diagnostic cohort for a sta6a, and the numerator is receipt of an EBP (simpler for EBP Reach Initiate). 

- I believe we are on track working to resolve the differences with EBP Course tho, right?

- I will look to follow up on this with you both as I land! It’s a relatively short flight to Seattle

**Comment by holbrooa** [comment]
@lzim 

- Roger

- Technically, neither. I mean this exact idea from the first sentence above: **X visits (with a template) in Y days**  We know they often work on multiple templates in the same visit, so we can't just count templates. And in fact that's why the policy definition specifies templates completed on different days. So I filter to visits with a template, then I count those visits.

- There's no penalty per se, but it helps to explain some potentially funny data. Like, "wait, they're doing all these templates...but they're neither starting nor completing with very many patients."  So the explanation is that they're either doing templates too slowly (probably the obvious explanation) or they're continuing treatment with their completers (the less obvious one that arises from this logic we're talking about). Or both.

- That's right. No penalty for not graduating completers.  If we really wanted to, we could calculate a sort of "EBP efficiency" by comparing completers to templates or something.

- They are. And I am too. This graduation concept is contained in the "8+ visits" part, before we get to the early termination part. We do this part first, presumably because of how costly it is to go looking for those test scores (that complete box must be backed up by the relevant test score, per jodie's scripts).

**Comment by lzim** [comment]
@holbrooa

This all sounds very on track to me. 

- good re: how handling templates and visits

- Right

- no ned for this now

- great 👍🏾 

If you have enough to proceed with this, then go for it 😃

**Comment by saveth** [comment]
@holbrooa I'm glad to hear that you got what you need to pull EBPsy Dose. Do you have enough info to operationalize the query for EBPharm Dose? If so, can you clarify for me on how we are counting EBPharm AUD and OUD?

**Comment by lzim** [comment]
For @saveth’s question, and given @holbrooa’s double-checking.

So, it sounds like removed EBP Course folks (to avoid double-counting in subsequent months) will be handled via removal from both numerator _and_ denominator (in subsequent months), to ensure an accurate EBP Course proportion for any given sta6a month.

**Comment by lzim** [comment]
BTW: Obviously still on the tarmac 😂

**Comment by saveth** [comment]
@holbrooa @lzim Based on the comments above, it sounds like I will be getting new numerators and new denominators variables for EBP Dose calculation, which is separate from the EBP Initiate calculation. Is this correct?

**Comment by holbrooa** [comment]
@saveth re: **AUD and OUD: are these the same people as those in the Initiate or are they a separate count, therefore new variables?** I'm not sure I'm following this question. If you mean new columns in the data table, then yeah. I will have new columns for each kind of completer. As for the same people, yes sort of. I mean, of course you have to get some of the drug (our initiate def) in order to get a therapeutic dose. But not necessarily in the same month. Or, looking at the depression one, definitely not in the same month.

**Comment by lzim** [comment]
> @holbrooa @lzim Based on the comments above, it sounds like I will be getting new numerators and new denominators variables for EBP Dose calculation, which is separate from the EBP Initiate calculation. Is this correct?

Yes, I believe so. Based on the discussion above, but also conceptually both the numerator and denominator have to be different for EBP Course

**Comment by holbrooa** [comment]
@saveth I'm trying to get EBPsy correct and to you first. Today, if at all possible (but these queries will take a long time). So I might have to come back to some of these ebpharm questions later. 

@lzim I'm not understanding the concept that causes the denominator to have to be different for EBP Course. Might have to follow up once you're back on the ground.

**Comment by saveth** [comment]
> @saveth re: **AUD and OUD: are these the same people as those in the Initiate or are they a separate count, therefore new variables?** I'm not sure I'm following this question. If you mean new columns in the data table, then yeah. I will have new columns for each kind of completer. As for the same people, yes sort of. I mean, of course you have to get some of the drug (our initiate def) in order to get a therapeutic dose. But not necessarily in the same month. Or, looking at the depression one, definitely not in the same month.

@holbrooa that's what I was originally thinking you'll produce. New columns with the EBP Dose counts of patients and we're using the same denominators (diagnostic cohort) for our reach. However, @lzim seems to suggests that how EBP Dose should be calculated will be different from how EBP Initiate was calculated, therefore requiring a different set of numerator and denominator for each month.

**Comment by staceypark** [comment]
it sounds like we need to hold until @lzim provides more clarification so @holbrooa can give the correct pieces to @saveth ?

**Comment by holbrooa** [comment]
My scripts are running. Fingers crossed. Re-reading this thread, I see now where I missed what @lzim was saying about the new denominators. I hadn't thought of removing those patients from the denominator. I'm trying to wrap my head around how that would work.

**Comment by lzim** [comment]
@saveth @holbrook @staceypark

On the ground at SEA!

It’ll be a minute before I get to a place where I could talk to any of you about this. 

But, where on a surface level (grant overview level), it makes sense to consider the exact same eligible diagnostic cohort denominator with either a...
1. EBP Reach Initiate numerator (to get a proportion/%) _or_ a...
2. EBP Reach Course numerator (to get a proportion/%)...

I just want to think through kind of lag issues described above related to EBP Course, that could potentially lead to under-counting or double-counting, etc.

**Comment by holbrooa** [comment]
Ok. Sounds like maybe we should talk about it tomorrow. For now, I've put a new data file in the folder for @saveth and I will add the pharm columns either tonight or in the morning, depending on whether or not my queries for that finish.

**Comment by lzim** [comment]
@saveth and @holbrooa

Okay, sure, thanks for all the work to clarify this today. It would be great for you to discuss at the meeting and document your further considerations/decisions in this issue. 

I can provide remote support💻, but only via online/text as I have SIRC responsibilities all day tomorrow.  

This is the question I’m considering with you both:

- Given how the query is run, if the _exact_ same diagnostic cohort denominator is used for EBP Reach _initiate_ and EBP Reach _Course_, _AND_ 

These two things are true:

> X visits (with a template) in Y days" type of logic

- BTW: Despite my confusing question, given all the units flying around (patients, visits, templates 🤓) I appreciated the extra detail you provided explaining how this was handled considering both visits _and_ templates.

For this, I added **bold** below: 
> I find the first time a patient hits that mark, i.e. the 8th visit in a span of 180 days or less, then throw out the visits after that so that you get counted once in the month where you "complete" **but then you don't count again in the following month.** E.g. a patient satisfies the above completion criterion for their visits 2 through 9, they get counted as a completer in the month that that 9th visit happened, and then I don't allow them to count again for visits 3 through 10, even though they technically satisfy the 8 visits in 180 days logic there.

- Despite our discussion of capturing EBPsy completers the month they complete, avoiding EBP _course_ “double-credit,” and the potential for “penalties” ...

- I am considering that if the eligible diagnostic cohort for a sta6a in any given month excluded prior completers _only_ from the numerator (but NOT the denominator), then it would _penalize a clinic in any given month that a previous “completer” continues in care (9th or more based on the quote above), is removed from the numerator, but is still in the denominator_ based on diagnosis.

- This type of concern brings up the purpose of aim 1, which is to tell us both how many patients get touched with an EBP at all (initiate), versus how many patients complete an adequate therapeutic course/dose adequate to meet their need.

- The monthly observation is a key part of our analyses, but arbitrary for capturing the conceptual purpose of aim 1 immediately above. 

- It would be clearer to explicitly translate the EBP Reach _Initiate_ numerator to become the EBP Reach _Course_ denominator, EXCEPT for the issue of “_in any given month_” and the lag between when EBP initiation and EBP Course are captured. 

- It was for these “lag” issues related to estimating an EBP Course proportion, in any given sta6a month, that would necessitate removing prior completers from both numerator and denominator, in order to avoid “penalizing” clinics for their prior completers.

**Comment by lzim** [comment]
Looping in @staceypark ☝🏽

**Comment by staceypark** [comment]
By "translate" do you mean we need another equation?
> It would be clearer to explicitly translate the EBP Reach Initiate numerator to become the EBP Reach Course denominator, EXCEPT for the issue of “in any given month” and the lag between when EBP initiation and EBP Course are captured.

**Comment by lzim** [comment]
@staceypark

 > @lzim By "translate" do you mean we need another equation?

No, this statement (also note the EXCEPTION) should’t be considered in isolation of the overall thread from today. 

Your quote from me is just one part of my multi-bullet attempt to describe what the potential issue is that I am attempting to clarify/resolve.

**Comment by lzim** [comment]
@holbrooa and @saveth 

I’m sorry about the delay responding to these questions...it was getting quite confusing to track with the level of **issue_incoming** I was fielding across the team yesterday.

> And since we're limiting ourselves to outpatient mh clinics, I don't think these other sources of meds are necessary.
- Other sources of meds for **OUD EBPharm** may indeed be key if they are receiving methadone at a methadone clinic, but otherwise have visits managed in VA outpatient care. For methadone, this is common, as there just aren’t that many methadone clinics.
- However, I do not expect this to be as common for **OUD Bupenorphine** or **AUD EBPharm**

> need our own definition or another source of a therapeutic dose of AUD and OUD ebpharm. I'm not even sure what we're looking for. Some kind of continuity threshold like we are using for the depression meds?
- Right, we do need something similar and we do have to create it ourselves.
- I have all my notes from Jodie’s recommendations, but unfortunately they are in my office in Menlo Park. However, @staceypark may be able to photograph them and send them to me for re-assessment. 
- I do believe she understood what we needed, and I believe she thought she could get it from the code she sent.
- If we do need to create our own definition, how can I help you propose/develop this ☺️

**Comment by lzim** [comment]
...working on thinking through the denominator issue now...

**Comment by lzim** [comment]
@holbrooa and @saveth 

- Agreed that this is the key issue that we are working to clarify:
> The question is: when do we count the completion, i.e. which month does a completer get captured in? 

- Our scientific audience reviewing the designs and results of a cluster randomized trial (CRT) are principally concerned with unbiased estimation of the difference between our two trial arms due to the intervention received in each arm (MTL vs. AF/uQI) - i.e., a phase III superiority CRT design. Therefore, considerations we have made for other parameter estimates (e.g., conserving staff time in models), are not primary here. 

-  Example of the expected description of tests of **specific aim 1 for omnibus EBP Reach Initiate:**
“The average omnibus (i.e. depression, OUD and PTSD) EBP Reach Initiate was 4% among MTL clinics during the twelve months pre-intervention and 4% in AF/uQI clinics for  during the 12 months pre-intervention. The average proportion of omnibus EBP Reach Initiate during the 12 months post-MTL intervention was 9% and remained 4% in the AF/uQI clinics. As hypothesized, the pre/post change in EBP Reach Initiate is a difference in proportions of 5% across trial arms (95% CI XX-XX) in .”

- will post more once I switch to next session

**Comment by holbrooa** [comment]
@lzim @saveth The example you give for expected description of tests makes a lot of sense. And it seems to me that that means we can't use option 2.

**Comment by lzim** [comment]
Busy with the conference and just getting a chance to turn back to this:

We’re considering **the role of the 8+ visits (with a template) in 180 days logic when we have 12-month average EBP reach observation periods**

- In specifying the **Specific Aims** analyses, I was trying to lift out the purpose of estimating a **12-month** period EBP reach estimate for 1) pre/post intervention, and 2) initiate and course.
- The _purpose_ of our R01/IIR trials is a clean, unbiased comparison of **MTL vs AF/uQI interventions**
- The _importance_ of accurate **monthly sta6a EBP reach observations** to enable the hypothesis tests according to specific aims, i.e., we do NOT want to use EBP reach measures that would make our trial arms incomparable because of when our EBP reach observation (Initiate or Course) is pulled **_relative to when trial interventions (MTL or comparator) occur in the clinics_.**

On the 3 **“completer definitions”** from @holbrooa’s example (I’ve added “EBP”)
1. the month of their first visit ("true EBP initiation") - our **aim 1a “EBP Reach Initiate” measure**
2. the month of their tenth visit ("EBP dose initiation") - 
3. the month of their 17th visits ("EBP completion") - this last visit seems closer to what we have operationalized as “graduation” in the PSY module of _Modeling to Learn_

- Sounds like Option 3 to me: We will want to capture when they first “qualify” for the EBP Reach _Course_ numerator (“dose initiation”) with the 12-month pre _OR_ 12-month post observation window
- And, they need to count toward the EBP Reach Course numerator for as long as they are engaged in care during the 12 month pre/post intervention period.

> @lzim @saveth The example you give for expected description of tests makes a lot of sense. And it seems to me that that means we can't use option 2.
- I agree that there seem to be many problems with option 2.

Quoting @saveth but LZ added _**emphasis**_
> depression patients _**newly treated**_ with an antidepressant medication

- Depression SAIL measures MDD43h (86 days) and MDD47h (180 days) are really _EBP Course_ measures in terms of how the grant conceptualizes it. However, _consistent with how SAIL is operationalized, and in terms of implementation science, the key idea is that the “evidence-based practice or EBP” is being evaluated for a therapeutic response, not a simple anti-depressant script/fill/release._ So, even though this is looking at care continuity, it is still an “initiate” measure.
- I’m not sure I’m understanding the rest of your post re: **adding** numerators over the same denominator? It seems it definitely would double-count them, unless I’m missing your proposal.  - - **Including in the numerator _any_ patients who meet the EBP measure - in any given sta6a month - _across the 12 month pre/post period of observation _ for any given sta6a month in which they are in the denominator_ is what we need.**

- Although there are differences across the 3 grants (R21, R01 and IIR) in monthly EBP Reach observation windows, and the diagnostic cohorts, we want to work for parallel EBPsy & EBPharm Reach _Course_ definitions _in terms of timing_ because each study design intends to determine whether clinics that get MTL have superior improvements in pre/post EBP reach to clinics that do not (R21 comparator clinics; R01 AF clinics, and IIR clinics)

**Comment by saveth** [comment]
@holbrooa @lzim  below is a clear explanation of the formulae I have in mind. 

***in Data file MDD43h = sail43h***
***in Data file MDD47h = sail47h***

For calculation of **SAIL MDD** formula is:

Na = number of patients who received 84 days of antidepressant medication
Nb = number of patients who received 180 days of antidepressant medication
Td = any patient diagnosed (dx) with depression (MDD) and newly treated with an antidepressant 

**MDD43h = Na/Td**
**MDD47h =Nb/Td**

**THEN**

**EBPharm Reach for MDD formula is**

Pd = in any given month is the number of patients diagnosed with MDD
Td = any patient dx with MDD and newly treated with an antidepressant (same Td from sail measure above)
Na = number of patients who received Rx for 84 day ( same Na from sail measure above)
Nb = number of patients who received Rx for 180 days (same Nb from sail measure above)

 **EBPharm Reach Initiate = Td /Pd** 
 **EBPharm Reach Dose =  (Na + Nb) /Pd** 

@lzim Based on your comment: "even though this is looking at care continuity, it is actually still an “EBP Initiate” measure", means that we still need to figure out how to pull the EBPharm Dose numerator and denominator for AUD, OUD, and Depression. 

Based on the explanation to Andrew's comments, it sounds like Andrew does need to pull a different set of denominators for EBPsy.

Based on my understanding for EBPsy:
*dx =  patients who are diagnosed with PTSD, Depression or AUD*
*Ts = True Initiation of EBPsy -- first visit*
*T8s = Dose Initiation- the start of an EBPsy dose treatment within six or 12 months of treatment*
*T8f = Dose Completion- completion of EBPsy treatment*

**EBPsy Reach Initiate = Ts/dx** (which is what we already have and calculated)
**EBPsy Reach Dose = T8f/T8s** (what Andrew still needs to pull)

and it sounds there needs to be two sets of T8s, one where dose initiation is within 12months and one where T8s is dose initiation is within 6months.

**Comment by lzim** [comment]
> @lzim Based on your comment: "even though this is looking at care continuity, it is actually still an “EBP Initiate” measure", means that we still need to figure out how to pull the EBPharm Dose numerator and denominator for AUD, OUD, and Depression.

- @saveth absolutely we do (see my post above from this morning)

**Comment by lzim** [comment]
> EBPharm Reach Initiate = Td /Pd

- Yes @saveth 



EBPharm Reach Dose = (Na + Nb) /Pd

**Comment by lzim** [comment]
So, based on my post above...

Depression EBPharm Reach Course = (Given Td, **Nb**/Pd)

- Pd = in any given month is the number of patients diagnosed with MDD
- Td = in any given month any patient dx with MDD and newly treated with an antidepressant (same Td from sail measure above)
- **Nb** = in any given month, number of patients who received Rx for 180 days (same Nb from sail measure above)

FYI: @saveth @holbrooa and @staceypark

**Comment by saveth** [comment]
@lzim Based on your clarification:

> Depression EBPharm Reach Initiate = (Given Td, **Na**)/Pd
> Depression EBPharm Reach Course = (Given Td, **Nb**/Pd)
> 
> * Pd = in any given month is the number of patients diagnosed with MDD
> * Td = in any given month any patient dx with MDD and newly treated with an antidepressant (same Td from sail measure above)
> * **Na** = in any given month, the number of patients who received Rx for 84 day ( same Na from sail measure above)
> * **Nb** = in any given month, number of patients who received Rx for 180 days (same Nb from sail measure above)

- we now have clarification of the EBPharm Reach for Depression for Initiation and Dose. Yay! 👍 
- Given the updated formula, you've provided, this means I'll need to to revised all the calculations and analyses I've done for EBP Reach Initiate involving EBPharm Depression. 

@lzim What about the formulae I've outlined for EBPsy Dose?

**Comment by lzim** [comment]
@saveth I keep trying to return to this, but it is a VERY busy conference for me. 💃🏻
FYI @holbrooa and @staceypark 

> Given the updated formula, you've provided, this means I'll need to to revised all the calculations and analyses I've done for EBP Reach Initiate involving EBPharm Depression.

- @saveth please do NOT revise all the calculations you have used **_for the purposes of site selection._**
- Let’s talk about this at Monday’s **Workgroup Leads Meeting** before you proceed with this conclusion.

- I want to clarfiy with @holbrooa your **Depression EBPharm** formulas above, particularly for your **Td** 
- This is what I see in **r21_r01_iir_procedures** on **OSF** here: https://osf.io/z7y8m/wiki/home/
> “Depression: Anyone who has filled a new antideppresant medication within 60 days of a visit. This matches the SAIL MDD43h and MDD47h denominator definition. Furthermore, the quality of EBPharm for depression, includes evaluation by a prescriber to assess whether the medication is achieving an intended therapeutic response.“

- For site selection, I do not want to CHANGE to this formula that we have been working on today, which is **to clarify EBP Reach _Course_ for the purposes of making sure we know how we will test specific aims, and can report this in our 1) R01 and IIR trial registrations, 2) R01/IIR protocols, and 3) R21 preliminary findings manuscript.**
- We will want to RETAIN the  _**current depression EBPharm Reach Initiate Calculation**_ for many reasons, and it is sufficient for site selection.
- However, the differentiation between **depression EBPharm _Initiate_ and _Course_** needs to be more precise, as described above, for _tests of specific aims_

_**Current depression EBPharm Reach Initiate Calculation**_
> EBPharm Reach Initiate = Td /Pd
> Pd = in any given month is the number of patients diagnosed with MDD
> Td = in any given month any patient dx with MDD and newly treated with an antidepressant (same Td from sail measure above)

**Comment by saveth** [comment]
@lzim 😌 Glad to hear we don't have to redo everything for site selection and that what we're discussing now is for analysis of the AIMS.

**Comment by lzim** [comment]
@saveth @holbrooa @staceypark 

Editing @saveth‘s formulas, which has several misunderstandings (or potential for misunderstanding) and also needs further discussion:
dx = in any given month, patients with PTSD, Depression or AUD on a visit in that sta6a
Ts = in any given month, “True Initiation of EBPsy” - **first EBPsy template on a visit, corrected by @holbrooa’s logic that addresses multiple templates in the same visit.**
_still TBD, but proposed_
T8s = Andrew’s “Dose Initiation” Lindsey’s **EBPsy Course** - **Denominator:** in any given month, patients with PTSD, Depression or AUD on a visit in that sta6a, who have had a minimum of **Numerator:** 8+ visits (with a template) within 180 days first EBPsy template on a visit, corrected by @holbrooa’s logic that addresses multiple templates in the same visit.** 

> and it sounds there needs to be two sets of T8s, one where dose initiation is within 12months > and one where T8s is dose initiation is within 6months.
- No. Nothing like this at all.
- The key from my post above is about the study design: we have pre/post 6-month, average monthly, EBP Reach observation windows (IIR) and pre/post 12-month, average monthly, EBP reach observation windows (R21 & R01) that have nothing to do with the “EBPsy completion” logic Andrew is working with from Jodie/OMHSP.

**Comment by holbrooa** [comment]
In the meeting, we decided to use Andrew's option 1 from above, where we count completers exactly once in the month where they first meet the completion criteria. This is how the current comparator dataset is, and the definitions are in the defs tab.

**Comment by saveth** [comment]
In the meeting (https://meet.lucidmeetings.com/lucid/meeting/224690 ), we've decided how w're calculating EBPsy and EBPharm Initiate is fine. For course/dose:

**EBPsy Course = EBPsy complete / EBPsy diagnostic cohort**, where
- EBPsy complete refers to the sta6a/month when the person complete the EBPsy treatment (they initiate dose  treatment within the prior 6 months)
- EBPsy diagnostic cohort is the unique patient count for a given sta6a/month

**EBPharm Course = (sail43hnum + oud84days + aud84days)/ EBPharm diagnostic cohort**, where
- sail43hnum is the number of depression-diagnosed patients who received >=84 days of antidepressant medication through 114 days after the prescription start date.
- oud84 days and aud84days will mimic sail43hnum with the difference of:
1. not seen for depresion within last year
2. a new anti-depressant scraip (i.e., has not had a prescription filled within last 105 days)
3. must have had an

**Comment by lzim** [comment]
DECISIONS MADE:

#1. We will use the "true initiation" or first eligible script or visit/template as our EBP Reach Initiate definition for any given sta6a month, as we have been.

- EBPharm - Denominators include diagnosis on visit OR eligible med.

- Depression EBPharm - 

(i.e., includes 3 unique criteria for the depression numerator not accounted for in OUD and AUD:

1. not seen for depresion within last year

2. a new anti-depressant scraip (i.e., has not had a prescription filled within last 105 days)

3. must have had an encounter  within 120 days of the prescription fill.)



#2. For EBPsy Reach Course we are counting only the month when a patient firsts meet the criteria worked out by OMHSP, for example X visits in Y days.

- The patient who continues after completing EBPsy is NOT removed from the denominator and is removed from the numerator in subsequent months.



#3. For EBPharm Reach Course we are paralleling SAIL MDD43h for AUD and OUD, without the special "new anti-depressant" rules applied (i.e., includes 3 unique criteria for the depression numerator not accounted for in OUD and AUD:

1. not seen for depresion within last year

2. a new anti-depressant scraip (i.e., has not had a prescription filled within last 105 days)

3. must have had an encounter  within 120 days of the prescription fill.)

- Andrew will include non-VA meds for OUD (e.g., Methadone Clinic).



#4. WE CONCLUDED HAVE TO USE ISSUE DATE TO TRACK REFILLS OF THE SAME PRESCRIPTION for the NUMERATOR

- SAME ISSUE DATE: for any refill goes back to original script, sum up supply, have to capture this in the month the prescription is issued.

- RELEASE DATE: in the denominator, captures a patient who got a script for that diagnosis in a given sta6a month.



#5. We decided NOT to work with MDD47h, but rather to work with MDD43h across all EBPharm.



NEXT STEPS

A.  Andrew will change the patcountDeprx, patcountAUDrx AND OUDRx to use Issue Date for the Numerator

1. Andrew will create the AUD and OUD EBP Course Numerators following MDD43h as a guide, but at the sta6a month level, and without applying the "depression exclusion."

2. Will use Issue Date across the EBPharm Numerators



B. Savet will add near the Latex formulae the formula that correpond to the specific columns/variables used in the table below.

- Work to clarify places where MDD43h actually means something specific at a sta6a month for our R01/IIR trails.

- Use MDD43h (not MDD47h as the EBPharm reference definition).



CONCLUSIONS

- Only one decision today impacts our R01/IIR Site Selection based on EBP Reach Initiate, and that is the use of Issue Date in the EBPharm numerator.

- We did not say that we would include EBP Reach Course in our Site Selection,

- We did not say we would use AUD EBP Reach in our Site Slection (in either the NIH or IIR)

- We are okay, continuing with our "True Initiate" EBPsy and EBPharm meaning that we've had with RELEASE date in the EBPharm Denominator.

**Comment by holbrooa** [comment]
Okay. There is a new comparator dataset. @saveth , please go check it out. It has the new columns we talked about, and I've updated the definitions tab.

**Comment by saveth** [comment]
@holbrooa Thanks! Can you also update the definitions on OSF (https://osf.io/z7y8m/wiki/Data%20Definition/) for your part? I'll update my part where it references your variables as I work on the dataset.

**Comment by saveth** [comment]
@holbrooa @lzim There's something wrong with how the new OUD initiate is being pulled. I'm getting reach over 100% for 8 clinics. I'll email the list of these sites.

**Comment by saveth** [comment]
@holbrooa There's a total of 45 sites where they are at or above 100% in OUD Reach.

FYI: @lzim @staceypark @ritahitching @saveth

**Comment by holbrooa** [comment]
@saveth How is this calculation done? I haven't found any mistakes in my script, so I guess I'll need to work backwards. Can you tell me where the 122% comes from for 655GC?

**Comment by lzim** [comment]
Thanks to you both for working on this!
GitHub Trackers are a useful space for project breakdown, clarification of next steps, and conceptual disucssion.

Reminder that detailed of any issues to resolve this must occur on our secure servers. 🔐 

Thanks!

**Comment by saveth** [comment]
@holbrooa I pinged you on OSF  (https://osf.io/z7y8m/wiki/Data%20Definition/) on how I'm calculating the reach and identified how the count of OUD initiate treated patient is higher than the OUD diagnostic cohort.

**Comment by saveth** [comment]
@holbrooa Thank you for looking into the fuzziness of the data. I agree with the solution of capping the average to 100% to deal with people being issued a prescription one month and picking it up the next month. This way we don’t have two sets of EBPharm denominator definitions. @lzim , do you agree?

With regards to how the average EBP measure is calculated, I’m trying to be consistent to how it has been stated in the granted on how we are doing the analysis. I believe the goal of looking at the average monthly EBP reach is to get a sense of, for a given month, how much of a reach is the site getting. Instead of saying for a given year, what is their overall reach. It is up to @lzim  if she would prefer to do a yearly average EBP reach instead of the average of 12 monthly EBP Reach. @lzim, do you want me to update my calculation as Andrew is proposing or stay the course on how we have already defined and implement the calculation?

FYI: @ritahitching @staceypark

**Comment by holbrooa** [comment]
@lzim @saveth @staceypark 

This question about the issue -> release and averaging computation is the last nagging issue before I can run the whole query for all the years to get @saveth what she needs. So then my part of https://github.com/lzim/teampsd/issues/615 and https://github.com/lzim/teampsd/issues/424 and https://github.com/lzim/teampsd/issues/476 will be done. (And https://github.com/lzim/teampsd/issues/672 if that's a legit issue.)

We added some documentation to osf about this, and of course all the different defs are there, but I guess we probably just need to briefly talk about it in the quant meeting? It shouldn't take a long time. I think we have a pretty clear options A, B, C. I could even try to type them out here if you want, instead of going and putting them in the meeting agenda.

**Comment by saveth** [comment]
@lzim We've answered your questions in OSF (https://osf.io/z7y8m/wiki/definition_details_needed/) during the quant meeting.  We still need an answer to our question in this tread, which we've posted in OSF (https://osf.io/z7y8m/wiki/Data%20Definition/) with, hopefully, enough detail to help you make a decision.

FYI: @holbrooa @saveth @ritahitching @staceypark

**Comment by lzim** [comment]
Great thank you!!!
@holbrooa @saveth @ritahitching @staceypark

To consider options A,B,C for OUD EBPharm (and likely AUD) we need definitions of 
**Issue date**:
and 
**Release date**:

Please see my edits to clarify on OSF:
https://osf.io/z7y8m/wiki/Data%20Definition/

**Comment by lzim** [comment]
Also FYI: @holbrooa @saveth @staceypark and @ritahitching 

Separate from us trying get to get the **Issue Date** and **Release Date** and therefore, the validity of the EBPharm Reach Initiate measure definitions crystal clear here: https://osf.io/z7y8m/wiki/Data%20Definition/

@saveth is correct, for a variety of additional proposed analyses, we need average monthly EBP Reach observations for each clinic of cluster size ~800 or larger, and not annual EBP reach.  
- EBP Reach is a proportion (%) in which the denominator refers to all actively engaged patients with eligible diagnoses in that clinic in a given month.
- The monthly eligible clinic population that is engaged in care denominator, is a dynamic, open population that requires the monthly temporal specificity rather than annual.

As we are doing, working together to think this all through carefully, clarifying distinctions between the average monthly period EBP reach and the annual period EBP reach is challenging for many reasons, including the need to account for both new and existing patients, different types of evidence-based care, care episodes of variable duration, differences in primary presenting concerns, as well as the possibility of continuing in care in ways that are no longer evidence-based.

We are working to assess the proportion of an eligible clinic population that is receiving EBPs at the same point in time across our arms. And, expect a more valid comparison, if our denominator and numerator are “refreshed” monthly as we have proposed.

I continue to welcome everyone’s best thinking about all of this, and very strong documentation.
Hopefully, we can pin down the last remaining questions on OSF and wrap this up.
Thanks for all your hard work and careful checking with each other and the team 🤗

**Comment by lzim** [comment]
@holbrooa @saveth @staceypark @ritahitching 

Response to what I’m seeing on OSF for **options A and B***
*information will hopefully still be forthcoming about the definition of “Issue Date” and “Release Date” to consider that Option C denominator question you are asking about

I believe this is what is being described as **option B:**
“Average the monthly percentages by summing the numerator and denominator, then dividing. We still would want to cap the result.”

Sum of total EBPs received over the period.
over the
Sum of total eligible patients over the period.

This first solution would be the population EBP reach (proportion). This would answer the question "What proportion of total eligible patients over the observation period received an EBP."
- But, it would ignore our need to capture the average of clustered clinic observations over time.

This is what is being described as **option A** on OSF:
“Average the monthly percentages the way it is currently doing it and just cap the result at 100%. This is computing monthly average reach percentages the same as a quantity: sum the values and divide by 12.“
- Unless this is underspecified, I’m not tracking why the 12 month divisor would be all that we need.

So, just to check, consider **Option D**: I believe that the approach we need is the mean clinic EBP reach (proportion). This answers the question "What was the average EBP reach (proportion) among eligible patients in each clinic month". 

We find the average EBP reach (proportions) in each clinic (sta6a) month, in which each clinic has a different number of eligible patients in any given month, and _therefore each EBP reach value estimated has a different denominator._

Yes, we want to sum the EBP reach proportions for each clinic, but _divide by the number of clinics_ as: 
((EBP Received in Clinic 1 in month 1/Eligible population in Clinic 1 in month 1)...(EBP Received Clinic n in month n/Eligible population Clinic n in month n))/n clinic months.

**Option B and D**methods will produce different results. Both would be valid, but have different interpretations.
- As I clarified above, _we need average monthly EBP reach for each clinic_, which I believe is **Option D**
- Unless we just need to better clarify what’s going on with **Option A**

Thanks! 👍

**Comment by lzim** [comment]
@holbrooa and @saveth

I found myself unsatisfied with the amount of time I had to respond to this issue yesterday.
- Although, the improved documentation on OSF really helps me get to the next layer of questions!
- I'll look for the "issue date" and "release date" answers I need on OSF.

But, I've done more thinking about the differing options, continuing to consider the role of the

1. Denominators, especially any small clinics who say reach one out of 2 patients with EBPs.
2. And, the differences we observe with the summing of numerators and dividing by the sum of denominators, versus averaging the proportions and dividing by the number of clinic observations.

It made me return to **Simpson's Paradox** and review the examples they provide for _kidney stone treatment_ and for _batting averages_

https://en.wikipedia.org/wiki/Simpson%27s_paradox

Example   1 |   |   |   |   |   |  
-- | -- | -- | -- | -- | -- | --
  | Jan | Feb | Mar | Summed Num/Summed Denom EBP Reach | Average EBP Reach for 3 Obs. |  
Clinic 1 | 12.5% (25/200) | 33.33% (30/90) | 100% (50/50) | 30.1% (25+30+50)/(200+90+50) | 48.6% (0.125+0.333+1)/3 |  
Clinic 2 | 47.1% (40/85) | 4% (12/300) | 48% (60/125) | 21.96% (40+12+60)/(85+300+125)*100 | 33% (0.471+0.04+0.48)/3 |  
Month Summed Num/Denom | 22.8% (25+40)/(200+85) | 10.8% (30+12)/(90+300) | 62.9% (50+60)/(50+125) | 25.5% (25+30+50+40+12+60)/(200+90+50+85+300+125) | 40.8%   (0.125+0.333+1+0.471+0.04+0.48)/6
Month Averaged | 29.8% (0.125+0.471)/2 | 18.7% (0.333 +0.04)/2 | 74% (1 + 0.48)/2 |   |   |  

Example   1 |   |   |   |   |   |  
-- | -- | -- | -- | -- | -- | --
  | Jan | Feb | Mar | Summed Num/Summed Denom EBP Reach | Average EBP Reach for 3 Obs. |  
Clinic 1 | 12.5% (25/200) | 33.33% (30/90) | 100% (50/50) | 30.1% (25+30+50)/(200+90+50) | 48.6% (0.125+0.333+1)/3 |  
Clinic 2 | 47.1% (40/85) | 4% (12/300) | 48% (60/125) | 21.96% (40+12+60)/(85+300+125)*100 | 33% (0.471+0.04+0.48)/3 |  
Month Summed Num/Denom | 22.8% (25+40)/(200+85) | 10.8% (30+12)/(90+300) | 62.9% (50+60)/(50+125) | 25.5% (25+30+50+40+12+60)/(200+90+50+85+300+125) | 40.8%   (0.125+0.333+1+0.471+0.04+0.48)/6
Month Averaged | 29.8% (0.125+0.471)/2 | 18.7% (0.333 +0.04)/2 | 74% (1 + 0.48)/2 |   |   |  

Notes. *We have this scenario. We have not required them to have EBPsy   in each month.
--
We also are talking about   small clinics that may either have 100% reach, or exceed 100% due to   issue/release imbalance.

Summary   of Observations:
--
1. The summed approach is consistently   lower than than the averaged approach.
2. The denominator (clinic size/eligible   patients) is unsuprisingly influential, with small clinic that have any EBPs (e.g., one patient receives EBPharm out of 2 served that month) inflating reach.

- I'm sorry to have to return to the **facilitate_workgroup** and **qual_workgroups** from now until about 12:30 or so.

- @ritahitching and @holbrooa I'm going to recommend that we postpone the co-I meeting to further consider these issues carefully.

Thanks All!

**Comment by holbrooa** [comment]
I'm working on adding more verbiage to osf. I don't see an appropriate place to add this particular nugget there, so I'll just add it here. @lzim the idea behind your first observation in your summary is really key:

"The summed approach is consistently lower than than the averaged approach, inappropriately ignores any clinics who have 0 EBP in the numerator, and inappropriately ignores the monthly variation we need."

...but you're wrong about that generalization and about which method is ignoring what. The summed approach is only lower because you picked the examples that way, with the biggest denominators lining up with the lowest percentages. See, when you average the percentages you throw away the information about the quantity, so 0/1 = 0/100 and 1/2 = 50/100 etc. For a quick counter-example, look at what happens if a clinic does 8/9 one month, and 0/1 the following month. Then you have 80% (summed) vs 44% (avg the pcts) This will be true for any example with different denominators in the different months. Maybe it's easier to see it by looking at the inverse: if a given clinic had the same denominator in every month, then the two methods are mathematically equal. The more the monthly denominators vary, the more the two methods will yield different results, with the summed version responding to the denominator variance. Like, to add on to that previous example, say the clinic has 0/991 in the second month instead of 0/1. The averaging the percentages method still says 44%, but now the summed version is 0.8% I guess the summed version looked to you like it was ignoring variation in the numerator, but it isn't. It's just also taking into account variation in the denominator, and your example denominators vary enough that that really affects the result.

Tl;dr: we're talking about weighted vs non-weighted averages, basically.

**Comment by holbrooa** [comment]
Oh, and if it isn't obvious, that throwing away of the quantity is where the >100% comes from. So image 2/1 in the first month and 1/3 in the second month.

**Comment by saveth** [comment]
@lzim @holbrooa One of the things that confuses me is that we don't time bound the numerator or have that individual counted towards the month when the individual was counted towards the denominator. Like the 2/1 example @holbrooa indicated. If the individual is not counted towards the numerator and denominator in the same month, then how can we interpret: of all folks in a given month, X received treatment. How is it possible that more people receive treatment relative to the diagnostic cohort that month? 

I think this discussion about average of monthly EBP Reach or average of yearly EBP Reach is us trying to untangle the epidemiological concepts of whether we're measuring prevalence or incidence. If we're measuring **prevalence**, which is proportion of cases in a given period of time, then the one-year period is a very good estimates of the diagnostic population who actually had initiated treatment. This requires us not to just sum across the numerator and denominator, but to ensure that they are unique individuals in the numerator and denominator. If they are counted towards the denominator in June and August, but only once for initiating treatment in June, then we are underestimating the prevalence of initiation in the patient cohort for the given year.

However, if we are trying to estimate the **incidence**, the rate of occurrence of an event or the risk of getting treated, the monthly proportion would be a good estimates. If we are trying to convey to our audience the chances of getting treatment in clinic A in a given month, it's the mean of our monthly rate for that clinic that would indicate how successful they are able to treat their pool of eligible patients in a given month.

FYI: @ritahitching @staceypark @saveth

**Comment by holbrooa** [comment]
Also, as for which method "ignores" things: 0 in the numerator isn't ignored by either option. But 0 in the denominator is significant. 
- The summing version will ignore 0/0. 
- The average percentage version will count 0/0 (the same as if it were 0/1 or 0/100).

**Comment by staceypark** [comment]
**Decisions** from: https://meet.lucidmeetings.com/lucid/meeting/228871
1. We will take empirical approach and review the data to what the impact of using Option B and and Cleaning up Option A code to remove 0's is.
2. Anytime there is a 0 in the denominator or when there is 0/0, that clinic cannot be part of the divisor.
3. We will present the results of Option A and Option B to co-I's for direction.

**EBPharm defending Issue/Issue** 
1. Questionable whether just getting refills without seeing a "doc" is really meeting the critera for "EBPharm" (i.e., "evidence-based")
2. Primarily because it's crisp/clean definition for the purposes of trial design (e.g., comparing two clinics in each arm on the same month).

**EBPharm Course/Dose**
- In month 12 of our observation window (this example is for the CY2018 Site Selection),  the query looks ahead 90 days to make sure that patients have had 84 or more days of medication supply on hand.

**Next Steps**
- Lindsey & Savet review the code to clarify how it is handling clinic months that have no eligible denominator.
- Andrew re-runs his query.
- Everyone reviews/updates the OSF Definitions baed on today's decisions.

**Comment by saveth** [comment]
@lzim @holbrooa The r function is working as I originally thought it would, it truly did remove the NAs from the calculations and only compute based on existing values. For instance:

month | denominator | numerator | proportion
-- | -- | -- | --
1 | NA | 4 | NA
2 | NA | NA | NA
3 | NA | NA | NA
4 | NA | NA | NA
5 | 1 | 2 | 2
6 | 1 | NA | NA
7 | NA | 1 | NA
8 | 3 | 2 | 0.6666667
9 | 1 | NA | NA
10 | NA | NA | NA
11 | 1 | 1 | 1
12 | NA | NA | NA

mean(data$proportion, na.rm = TRUE) # answer is 1.22

Based on today's discussion, the change I would need to make to my code for our data is to replace all NA values in the numerator where there is a denominator,  so that the monthly proportion for that month is a true 0. Such that the example would look like:

month | denominator | numerator | proportion
-- | -- | -- | --
1 | NA | 4 | NA
2 | NA | NA | NA
3 | NA | NA | NA
4 | NA | NA | NA
5 | 1 | 2 | 2
6 | 1 | 0 | 0
7 | NA | 1 | NA
8 | 3 | 2 | 0.6666667
9 | 1 | 0 | 0
10 | NA | NA | NA
11 | 1 | 1 | 1
12 | NA | NA | NA

mean(data$proportion, na.rm = TRUE) # answer is 0.73

which will reduce the average proportion by including the true 0s.

**Comment by saveth** [comment]
@lzim @staceypark @ritahitching 
> 1. We will take empirical approach and review the data to what the impact of using Option B and and Cleaning up Option A code to remove 0's is.
> 2. Anytime there is a 0 in the denominator or when there is 0/0, that clinic cannot be part of the divisor.
> 3. We will present the results of Option A and Option B to co-I's for direction.

Are we expecting a second set of analysis similar to the one generated under current conditions (heatmap, boxplot, tree analysis, etc.)? 

FYI: @holbrooa

**Comment by saveth** [comment]
Hey @holbrooa, we were talking today during lunch, and I'm given the impression that our denominators are counted as follow: 

**Denominator**
**Omnibus EBP**: anyone with a visit or a prescription
**EBPsy**: we're counting people who have a visit with one of the relevant primary diagnosis
**EBPharm**: we're only counting people who have a prescription issued

If this is the case, then it would lead to the problem where EBPharm Reach becomes 1issue/1issue = 100% in any instance when prescription issued occurs. 

However, based on the OSF (https://osf.io/z7y8m/wiki/Data%20Definition/) the EBPharm denominator includes two types of people: 1) people who had a visit with the diagnosis or 2) receive a prescription.  @holbrooa, can you clarify if our denominator definition in OSF is correct or the one @lzim discussed during lunch is our updated definition based on the meeting this morning?

FYI: @staceypark @ritahitching @saveth

**Comment by holbrooa** [comment]
The one in OSF is correct. Prescriptions are used as a supplement to visits in the denominator. Every denominator includes visits with a relevant diagnosis. We're changing how we're doing that supplementation, is all. Visits have always been the main source of our denominators, and they will continue to be.

**Comment by holbrooa** [comment]
As you correctly point out, it'd be pretty silly to stop using visits for the ebpharm denominator, because then our definitions for the initiate numerator and denominator would be the same.

You could look at it like this: Patients who get a visit with the relevant diagnosis make up the denominator. But, since a new prescription will get you into the numerator, and it is sometimes possible to get a script without a visit, we should go ahead and count you in both the numerator and the denominator, even with no visit.

**Comment by staceypark** [comment]
@holbrooa want to clarify that I'm reading the OSF definition correctly because there's a potential question that @ritahitching was going to raise...
the denom counts 1) patients with dx who get visits with script, 2) patients with dx who get visit without the script, 3) patient who gets script without visit, 4) and anyone who already has a script (and we know they have it because of the issue date)

**Comment by holbrooa** [comment]
@saveth There's another version of the data table to look at. Only the BaseAndRx tab is different, because I didn't rerun the ebpsy stuff. Oh, and I updated the definitions, but not in osf yet, so I'll do that next.

@staceypark I don't really understand what you're saying. I think your number 4 is what we just decided not to include anymore in that last meeting, but I haven't updated the osf defs yet. Anyway, what's @ritahitching 's question?

**Comment by ritahitching** [comment]
@holbrooa thanks for offering to clarify

I was trying to understand how a given patient with more than 1 medication script (e.g. antidepressant and anti-anxiety medication) is counted in the denominator over time, as some prescriptions typically have a pharmacist release/issue date of 3 months before refill prescription is needed (e.g. 90 pills of supply of bupropion) and others 1 month supply (e.g.alprazolam) - unrelated to dosage prescribed. 

FYI @staceypark @saveth @lzim

**Comment by saveth** [comment]
@holbrooa Thanks! @saveth I'm changing my code to include what we discussed yesterday:

- [ ] for NULL/denominator value = 0/number (e.g. Null/ 2 = 0/2)  such that for average monthly proportion this will be 0 instead of NULL (0/2 =0 instead of NULL/2 = NULL)
- [ ] Create second deck of analysis using the average = sum monthly numerator/ sum monthly denominator
* The later part will add a substantial amount of time (about a week to the two days estimates for the first part of updating current analysis ) as I'll need to generate another set of 12 scripts to handle the different batch of EBP analysis

I will need @lzim  input next week to discuss how we should incorporate the new analysis into the co-I slide deck once we have a sense of what data looks like.

🤞 That this round of data validation will go well and our definitions and analysis will be finalized.

**Comment by holbrooa** [comment]
@ritahitching 

That's a good question. Inscrutable wall-o-text incoming...

Thankfully, now that we're counting original prescriptions everywhere, the situation is much simpler than before. When a patient gets one of the prescriptions we're tracking, we count them in the relevant denominator in the month that the prescription was issued. If they receive multiple tracked prescriptions, we count them in multiple denominators, but there are separate columns of combined denominators where they won't be double-counted. Refills aren't counted for our denominators anymore. They're only used in the numerator now, where we aggregate on each prescription to see how many days' worth of medication a patient received within a 115 day window to see if they have a steady supply.  So in terms of counting in the denominator over time, if they have to come back for a new prescription, then that means that the clinic's doctors are getting involved again and we re-count them in the month that the new prescription is issued.  I think you are right about some drugs requiring more frequent involvement from the prescriber, via limits on numbers of refills or days of supply or whatever, but from the perspective of what we're trying to count as I understand it, I believe we're doing the correct thing now: when the prescriber gets involved again, count that patient again in the denominator. I.e. a patient coming in to the pharmacy for a routine refill isn't an opportunity for the clinic to do evidence-based care. But the patient coming in to the clinic to see the prescriber who writes a new prescription is such an opportunity.  Have caution taking that logic to the bank though; I made this case months ago to use issue date in the denominator and Lindsey heard me out but went with release instead, so I may very well be misunderstanding something here.

Looking at your example, bupropion is a medication we're tracking for depression, but alprazolam isn't in our list. Any medications that we're not tracking are invisible to us - the patients we're tracking might be getting any number of other scripts, but we're just not pulling that information. But if a patient was issued a prescription for methadone in addition to bupropion in the same clinic in the same month (I have no idea if that's medically appropriate, but just exempli gratia) then they'll get counted once in each of a bunch of different denominator columns: 
- The depression diagnostic cohort for pharmacotherapy
- The depression diagnostic cohort for any of pharmaco- or psycho-therapy
- The opioid use diagnostic cohort
- The omnibus cohort for any pharmacotherapy
- The omnibus cohort for any pharmacotherapy that excludes alcoholism
- The total cohort 
- The total cohort that excludes alcoholism

...understanding all of that is probably like the text version of hitting a home run to left field here in Boston.

**Comment by ritahitching** [comment]
@holbrooa thanks for the clarification - I believe it's a home run on my end.

**Comment by staceypark** [comment]
@holbrooa will get @saveth data for #615 by next Tues

**Comment by saveth** [comment]
@holbrooa We were looking at the comparator data this morning, and I need more clarification on some of the variables you've pulled so that I can better explain to @lzim the calculations based on these variables. The questions are posted on OSF: https://osf.io/z7y8m/wiki/Data%20Definition/

FYI: @saveth @staceypark @ritahitching

**Comment by saveth** [comment]
@holbrooa Have you had a chance to look at the questions on OSF? We are trying to figure out which of the columns to use for our sample size calculations.

FYI: @lzim @staceypark @ritahitching @saveth

**Comment by holbrooa** [comment]
Yeah, I just posted some answers. In general, the information is all already in the definitions of each column, but I'll work on a narrative description of the queries as part of my documentation these next couple weeks.

**Comment by saveth** [comment]
@holbrooa For each of the EBPsy diagnostic cohort, we include people  with only a visit PTSD and Depression. Yet, for EBPsy AUD, we include people with prescription, and we use this for the denominator for EBPsy and EBPharm. Should we have two AUD diagnostic cohort? One with just the diagnosis for the EBPsy and one with the diagnosis + prescription for EBPharm?

FYI: @ritahitching @staceypark @lzim @saveth

**Comment by staceypark** [comment]
@holbrooa 
Making sure that I get your answer, on bullet three when you say "Once each in every denominator that includes one of the above" do you mean:

a) denominator includes any of the criteria: counted 1x regardless of how many criteria it suffices
b) same scenario: counted 1x per criteria i.e. if denominator includes all 4 criteria then it would be counted 4x since it has all of it

seems like the answer would be A because of our "no double counting" principle. but the word "each" is throwing me off a little.

> Are clinicians able to record multiple treatments on a visit? Can a patient go in for an EBPharm prescription and an EBPsy PTSD? If so, how are they counted towards OmniPsyCount, OmniPharmCount, TotalCohort, EBP Templates and DepPharm84Denom or DepPharm84Num?

> So if a patient has a visit where the provider marks a diagnosis of PTSD and also a diagnosis of depression, and the patient gets some depression meds and gets evidence-based psychotherapy for their PTSD, then they are counted:

> Once in the denominator for depression
> Once in the denominator for PTSD
> Once each in every denominator that includes one of the above
> Once in the EBPsy template numerator for whichever template the provider used
> Once in the depression meds numerator

**Comment by staceypark** [comment]
@saveth (cross ref #476)
In-preparation for the Co-I meeting on 10/22:
1. By COB Tuesday 10/15 detail all the changes you intend to make to the slidedeck 
2. By COB Thursday 10/17 ping @ritahitching and @staceypark with updated slidedeck. We will help review for clarity & typos before sending it to @lzim 
3. By COB Friday 10/18 send slidedeck to @lzim for final review to discuss and present to co-I's at Tuesday 10/22 meeting

Note: @lzim will be out most of next week for AHSR.

**Comment by staceypark** [comment]
cross-ref to @saveth handoff #615 
@saveth has database for course going back to 2014
